config/webpack/loaders/devSitePropsTable.js

```diff
@@ -0,0 +1,134 @@
+const { getOptions } = require('loader-utils');
+const mdx = require('@mdx-js/mdx');
+const reactDocs = require('react-docgen');
+const findSource = require('../loaderUtils/findSource');
+
+/**
+ * Create an inline mdx react component.
+ * @param {*} description
+ * @param {*} callback
+ */
+const propMdx = async function propMdx(description, options, callback) {
+  const mdxOptions = {
+    ...options,
+    filepath: this.resourcePath,
+    skipExport: true,
+  };
+
+  try {
+    const code = [
+      '() => {',
+      await mdx(description, mdxOptions),
+      // execute the mdx function returned.
+      'return MDXContent({});',
+      '}',
+    ].join('\n');
+    return code;
+  } catch (err) {
+    return callback(err);
+  }
+};
+
+/**
+ * Create the type mdx react component
+ * @param {name, value} type
+ */
+const propType = (type, options, callback) => {
+  const { name } = type;
+  let { value } = type;
+
+  if (type.name === 'enum') {
+    // strip single quotes.
+    value = Object.values(value).map((obj) => obj.value.replace(/'/gi, ''));
```

Comment:
is the case insensitive modifier necessary?

Reply:
no, it isn't